sdk/communication/azure-communication-chat/README.md

```diff
@@ -0,0 +1,714 @@
+# Azure Communication Chat Service client library for Android
+This package contains the Android SDK for Azure Communication Services for Chat.
+Read more about Azure Communication Services [here](https://docs.microsoft.com/azure/communication-services/overview).
+
+# Getting started
+
+## Prerequisites
+
+* You must have an [Azure subscription](https://azure.microsoft.com/free/) to use this library.
+* An Azure Communication Resource, learn how to create one from [Create an Azure Communication Resource](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource).
+* The client libraries natively target Android API level 21. Your application's minSdkVersion must be set to 21 or higher to use this library.
+* The library is written in Java 8. Your application must be built with Android Gradle Plugin 3.0.0 or later, and must be configured to [enable Java 8 language desugaring](https://developer.android.com/studio/write/java8-support.html#supported_features) to use this library. Java 8 language features that require a target API level > 21 are not used, nor are any Java 8+ APIs that would require the Java 8+ API desugaring provided by Android Gradle plugin 4.0.0.
+
+### Versions available
+The current version of this library is **1.0.0-beta.1**.
+
+> Note: The SDK is currently in **beta**. The API surface and feature sets are subject to change at any time before **GA**. We do not currently recommend them for production use.
+
+### Install the library
+To install the Azure client libraries for Android, add them as dependencies within your
+[Gradle](#add-a-dependency-with-gradle) or
+[Maven](#add-a-dependency-with-maven) build scripts.
+
+#### Add a dependency with Gradle
+To import the library into your project using the [Gradle](https://gradle.org/) build system, follow the instructions in [Add build dependencies](https://developer.android.com/studio/build/dependencies):
+
+Add an `implementation` configuration to the `dependencies` block of your app's `build.gradle` or `build.gradle.kts` file, specifying the library's name and the version you wish to use:
+
+```gradle
+// build.gradle
+dependencies {
+    ...
+    implementation "com.azure.android:azure-communication-chat:1.0.0-beta.1"
+}
+
+// build.gradle.kts
+dependencies {
+    ...
+    implementation("com.azure.android:azure-communication-chat:1.0.0-beta.1")
+}
+```
+
+#### Add a dependency with Maven
+To import the library into your project using the [Maven](https://maven.apache.org/) build system, add it to the `dependencies` section of your app's `pom.xml` file, specifying its artifact ID and the version you wish to use:
+
+```xml
+<dependency>
+  <groupId>com.azure.android</groupId>
+  <artifactId>azure-communication-chat</artifactId>
+  <version>1.0.0-beta.1</version>
+</dependency>
+```
+
+## Create the AzureCommunicationChatClient
+
+Use the `AzureCommunicationChatServiceAsyncClient.Builder` to configure and create `AzureCommunicationChatClient`.
```

Comment:
```suggestion
Use the `AzureCommunicationChatServiceAsyncClient.Builder` to configure and create an `AzureCommunicationChatClient`.
```

Reply:
This one is a tough call. If we are talking about class, without "an" makes sense. If we want to talk about object instance, "create an instance of AzureCommunicationChatClient ".